src/bento/dashboardTabData.js

```diff
@@ -910,8 +910,279 @@ export const GET_FILE_IDS_FROM_FILE_NAME = gql`
       }
   }`;
 
+export const GET_TREATMENT_OVERVIEW_QUERY = gql`
+query treatmentOverview(
+    # Demographics
+    $participant_ids: [String],
+    $ethnicity: [String],
+    $race: [String],
+    $sex_at_birth: [String],
+
+    # Diagnoses
+    $age_at_diagnosis: [Int],
+    $anatomic_site: [String],
+    $diagnosis: [String],
+    $diagnosis_classification_system: [String],
+    $diagnosis_basis: [String],
+    $disease_phase: [String],
+
+    # Studies
+    $dbgap_accession: [String],
+    $study_acronym: [String],
+    $study_name: [String],
+
+    # Survivals
+    $age_at_last_known_survival_status: [Int],
+    $first_event: [String],
+    $last_known_survival_status: [String]
+
+    # Table config
+    $first: Int,
+    $offset: Int,
+    $order_by: String,
+    $sort_direction: String
+) {
+    treatmentOverview(
+        # Demographics
+        participant_ids: $participant_ids,
+        ethnicity: $ethnicity,
+        race: $race,
+        sex_at_birth: $sex_at_birth,
+
+        # Diagnoses
+        age_at_diagnosis: $age_at_diagnosis,
+        anatomic_site: $anatomic_site,
+        diagnosis: $diagnosis,
+        diagnosis_classification_system: $diagnosis_classification_system,
+        diagnosis_basis: $diagnosis_basis,
+        disease_phase: $disease_phase,
+        
+        # Studies
+        dbgap_accession: $dbgap_accession,
+        study_acronym: $study_acronym,
+        study_name: $study_name,
+
+        # Survivals
+        age_at_last_known_survival_status: $age_at_last_known_survival_status,
+        first_event: $first_event,
+        last_known_survival_status: $last_known_survival_status
+
+        # Table config
+        first: $first,
+        offset: $offset,
+        order_by: $order_by,
+        sort_direction: $sort_direction
+    ) {
+        # Participant
+        participant_id
+
+        # Treatment
+        treatment_id
+        age_at_treatment_start
+        age_at_treatment_end
+        treatment_type
+        treatment_agent
+        dbgap_accession
+        __typename
+    }
+}`;
+export const GET_TREATMENT_RESPONSE_OVERVIEW_QUERY = gql`
+query treatmentResponseOverview(
+    # Demographics
+    $participant_ids: [String],
+    $ethnicity: [String],
```

Comment:
treatmentResponseOverview and treatmentOverview queries need to be updated from the new BE changes

Reply:
Resolved.